### DISCUSSION TITLE
Footnotes: What are illegal characters in keys?

### DISCUSSION THREAD

**Discussion by nschloe** [community/community#18842] [Other Feature Feedback, Questions, & Ideas]
It appears that one cannot use all characters in GitHub Markdown footnotes. By trial and error, I found the following not to work (or work incompletely):

- `A-Z` (links don't work) (perhaps a bug, see https://github.com/github-community/community/discussions/18798)
- `:`
- `[`
- `]`
- `|`
- `\`
- `<`
- `>`

Not sure if complete. It'd be nice if this was documented somewhere.

**Comment by dmgardiner25** [comment]
@nschloe is this issue specific to Codespaces and if so, could you describe how?

Here are the docs on footnotes for reference: https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#footnotes

**Reply by nschloe** [reply]
Nope, wrong category. It's everywhere.
